content/security/rate-limiting.md

```diff
@@ -35,13 +35,47 @@ Once the module has been imported, you can then choose how you would like to bin
 
 There may be a time where you want to bind the guard to a controller or globally, but want to disable rate limiting for one or more of your endpoints. For that, you can use the `@SkipThrottle()` decorator, to negate the throttler for an entire class or a single route. The `@SkipThrottle()` decorator can also take in a boolean for if there is a case where you want to exclude _most_ of a controller, but not every route.
 
-There is also the `@Throttle()` decorator which can be used to override the `limit` and `ttl` set in the global module, to give tighter or looser security options. This decorator can be used on a class or a function as well. The order for this decorator does matter, as the arguments are in the order of `limit, ttl`.
+```typescript
+@SkipThrottle()
+@Controller('users')
+export class UsersController {}
+```
+
+This `@SkipThrottle()` decorator can be used to skip a route or a class or to negate the skipping of a route in a class that is skipped.
+
+```typescript
+@SkipThrottle()
+@Controller('users')
+export class UsersController {
+  // This route will skip rate limiting.
+  @SkipThrottle(false)
+  dontSkip() {
+    return "List users work with Rate limiting"
+  }
+
+  // Rate limiting is applied to this route.
+  doSkip() {
+    return "List users work without Rate limiting"
+  }
```

Comment:
I think the comments are backwards here. `@SkipThrottle(false)` _should_ apply rate-limiting, right?

Reply:
Yes maybe I confused the comments, so if I understand correctly when `@SkipThrottle(false)` is not skipped the rate limiting only when and `@SkipThrottle(true)` right? 

P.s. would you like to send me the change request for this part 😁

Reply:
The `@SkipThrottle()` has a default `true` value, hence when you use it you apply the skip metadata. When you use `@SkipThrottle(false)` you are negating a previously applied skip (like wanting to skip for most of a contender but not a certain route).

Reply:
Thanks so much for the explanation, it will be useful to me  😉👍.

P.s. would you like to send me the change request for this part 😁

Reply:
Should I update this part like this:
```typescript
  // Rate limiting is applied to this route.
  @SkipThrottle(false)
  dontSkip() {
    return "List users work with Rate limiting"
  }
  // This route will skip rate limiting.
  doSkip() {
    return "List users work without Rate limiting"
  }
```

right?

Reply:
@jmcdo29, I updated the code snippets. Let me know if I need to make any other changes 👍